openmrs/apps/documentUpload/app.json

```diff
@@ -8,6 +8,6 @@
       ],
       "contextModel": [],
   "config": {
-    "preferExtraIdInSearchResults": true
+    "preferExtraIdInSearchResults": false
```

Comment:
This we can set it to true, the search can happen on all ID's.
We need to display only the Primary ID.